.github/ktlint.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is pretty cool, didn't know that GitHub allows you to elevate certain parts of the build output!